addons/arsenal/functions/fnc_attributeAddCompatible.sqf

```diff
@@ -51,7 +51,7 @@ if (_category == 8) then {
             _itemsToAdd append _magazines;
 
             {
-                _itemsToAdd append ([_magazineGroups, toLower _x] call CBA_fnc_hashGet);
+                _itemsToAdd append ([_magazineGroups, _x] call CBA_fnc_hashGet);
```

Comment:
You cannot rely on these names being config case correct.
If you make this case sensitive, the same case should be sued everywhere. Best way to do that is to make them config case like is already done for the magazines here
`_magazines = _magazines apply {configName (_cfgMagazines >> _x)};`

Reply:
Already done in fnc_scanConfig.

Reply:
No not done.
_x comes from `} forEach getArray (_muzzleConfig >> "magazineWell");`